.github/workflows/push.yml

```diff
@@ -0,0 +1,13 @@
+name: Pre-commit hooks
+
+on: [push]
+
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
There is a new major version: https://github.com/actions/checkout/releases/tag/v4.1.1